examples/Loading and preprocessing DSGE data.ipynb

```diff
@@ -0,0 +1,717 @@
+{
```

Comment:
<p>I like the heatmaps you use in the other notebook better than just a table, so the viewer's eye can follow the colormap. Personally, blocks of numbers just wash over me like I'm on ketamine.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jessegrabowski/gEconpy/pull/33/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>